src/styles/index.ts

```diff
@@ -3587,31 +3587,19 @@ const styles = (theme: ThemeColors) =>
             flex: 1,
         },
 
-        searchPressable: {
-            height: variables.componentSizeNormal,
-        },
-
-        searchContainer: {
```

Comment:
I search in all project files using IDE search and found no usages of this. I believe this are leftovers after some temporary old search feature which was removed in the end